bin/improver-nowcast-extrapolate

```diff
@@ -52,7 +52,7 @@ def main():
 
     group = parser.add_mutually_exclusive_group()
     group.add_argument("--output_dir", metavar="OUTPUT_DIR", type=str,
-                       help="Directory to write output files.")
+                       default="", help="Directory to write output files.")
```

Comment:
I've added this because this default already exists within the `improver-nowcast-optical-flow` CLI, which takes similar options, and as I generally interpret a variable beginning with `--` in a CLI to be optional, then I would expect this option to have a default. The result is that the output file for this CLI will be written into the current working directory.